Slides/Iterate-slide-elements/.NET/Iterate-slide-elements/Program.cs

```diff
@@ -0,0 +1,155 @@
+                    else if (shape.AutoShapeType == AutoShapeType.Rectangle)
+                    {
+                        shape.SetHyperlink("www.google.com");
+                    }
+                    break;
+                }
+
+            case SlideItemType.ConnectionShape:
+                {
+                    IConnector connector = shape as IConnector;
+                    // Modify the arrowhead style at the beginning of the connector line
+                    connector.LineFormat.BeginArrowheadStyle = ArrowheadStyle.ArrowDiamond;
+                    break;
+                }
+
+            case SlideItemType.Placeholder:
+                {
+                    // Modify text if present in the placeholder
+                    if (!string.IsNullOrEmpty(shape.TextBody.Text))
+                    {
+                        ModifyTextPart(shape.TextBody);
+                    }
+                    break;
+                }
+
+            case SlideItemType.Picture:
+                {
+                    // Resize the picture
+                    IPicture picture = shape as IPicture;
+                    picture.Height = 160;
+                    picture.Width = 130;
+                    break;
+                }
+
+            case SlideItemType.Table:
+                {
+                    // Get the table shape
+                    ITable table = shape as ITable;
+
+                    // Iterate through rows and modify text in each cell
+                    foreach (IRow row in table.Rows)
+                    {
+                        foreach (ICell cell in row.Cells)
+                        {
+                            ModifyTextPart(cell.TextBody);
+                        }
+                    }
+                    break;
+                }
+
+            case SlideItemType.GroupShape:
+                {
+                    // Get the group shape and iterate through child shapes
+                    IGroupShape groupShape = shape as IGroupShape;
+                    foreach (IShape childShape in groupShape.Shapes)
+                    {
+                        ModifySlideElements(childShape, presentation);
+                    }
+                    break;
+                }
+
+            case SlideItemType.Chart:
+                {
+                    // Modify chart properties
+                    IPresentationChart chart = shape as IPresentationChart;
+                    chart.ChartTitle = "Purchase Details";
+                    chart.ChartTitleArea.Bold = true;
+                    chart.ChartTitleArea.Color = OfficeKnownColors.Red;
+                    chart.ChartTitleArea.Size = 20;
+                    break;
+                }
+
+            case SlideItemType.SmartArt:
+                {
+                    // Modify SmartArt content
+                    ISmartArt smartArt = shape as ISmartArt;
+                    ISmartArtNode smartArtNode = smartArt.Nodes[0];
```

Comment:
@DharanyaSakthivel-SF4210 - Can you please iterate through the SmartArt Nodes and pass the `smartArtNode.TextBody` to `ModifyTextPart(ITextBody textBody)` method?